README.md

```diff
@@ -28,58 +28,11 @@ feature requests, and contribute with pull requests, if possible.
 ## Requirements
 - [Node.js](https://nodejs.org/en/) 10.0 or above
 
-## Setting up the Environment
+## llama-pack Components
 
-### Get the Java Development Kit (JDK) 8.
-The Android Command line tools requires the correct version of the JDK to run. To prevent version
-conflicts with a JDK version that is already installed, llama-pack uses a JDK that can unzipped in
-a separate folder.
-
-Download a version of JDK 8 that is compatible with your OS from
-[AdoptOpenJDK](https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=hotspot)
-and extract it in its own folder.
-
-**Warning:** Using a version lower than 8 will make it impossible to compile the project and higher
-versions are incompatible with the Android command line tools.
-
-### Get the Android command line tools
-Download a version of Android command line tools that is compatible with your OS from
-[https://developer.android.com/studio#command-tools](https://developer.android.com/studio#command-tools).
-Create a folder and extract the downloaded file into it.
-
-### Tell llama-pack where the JDK and Android command line tools are
-When running `llama-pack` for the first time, it will ask where it can find the JDK and Android command
-line tools. So, take note of the location where both were decompressed.
-
-
-## Using llama-pack
-### Initializing an Android Project
-Generate an Android project from an existing Web Manifest:
-
-```shell
-npx GoogleChromeLabs/llama-pack#0.3.0 init --manifest https://my-twa.com/manifest.json
-```
-
-When initalizing a project, llama-pack will download the Web Manifest and ask you to confirm
-the values that should be used when building the Android project.
-
-It will also ask you for the details needed to generate a signing key, used to sign the
-app before uploading to the Play Store.
-
-### Building the Android Project
-```shell
-npx GoogleChromeLabs/llama-pack#0.3.0 build
-```
-
-When building the project for the first time, the Android Build Tools will need to be installed.
-The tool will inkove the installation process for the build tools. Make sure to read and accept
-the license agreement before proceeding.
-
-As a result of the build step, the tool will generate a signed APK (`app-release-signed.apk`)
-that can be uploaded to the Play Store. You will also need to deploy a Digital Asset Links file to
-validate your domain. The
-[TWA Quick Start Guide](https://developers.google.com/web/updates/2019/08/twas-quickstart#creating-your-asset-link-file)
-explains how to extract the information needed to generate it.
+- **[llama-pack-core](./packages/core):** a javascript library that for generating, building and
+updating TWA projects.
+- **[llama-pack-cli](./packages/cli):** a command-line version for llama-pack.
```

Comment:
A command-line "version" sounds a bit odd.

"llama-pack core wrapped for the command line" maybe?

Reply:
"command-line interface for llama-pack"?